crates/aptos/src/test/mod.rs

```diff
@@ -591,13 +593,15 @@ impl CliTestFramework {
             entry_function_args: EntryFunctionArguments {
                 function_id: MemberId::from_str("0x1::staking_contract::create_staking_contract")
                     .unwrap(),
-                args: vec![
-                    ArgWithType::address(self.account_id(operator_index)),
-                    ArgWithType::address(self.account_id(voter_index)),
-                    ArgWithType::u64(amount),
-                    ArgWithType::u64(commission_percentage),
-                    ArgWithType::bytes(vec![]),
-                ],
+                arg_vec: ArgWithTypeVec {
+                    args: vec![
+                        ArgWithType::address(self.account_id(operator_index)),
+                        ArgWithType::address(self.account_id(voter_index)),
+                        ArgWithType::u64(amount),
+                        ArgWithType::u64(commission_percentage),
+                        ArgWithType::bytes(vec![]),
+                    ],
```

Comment:
We should probably add a unit test for this function now (I feel like I've avoided it too long), that checks the multi-nested BCS, and ensures it can be deserialized as the type expected.

Reply:
Agreed. I am also working on a PR that will extend this functionality to include JSON input files, and it looks like it'll involve some minor refactors to struct nesting. Once we can get that in perhaps it'll be appropriate to add more unit testing?